modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation to the cluster.
+
```

Comment:
IMHO: I would recommend to move this to the end of the customization doc here. So giving the reader first an introduction how to configure the available options, and then with this hint you can fine-tune the displayed options in the dropdown. Wdyt?

Reply:
Thanks for the suggestion but as per the docs consistency, we define or explain the "what" part with the action item then the proper how-to procedure. This is a kind of template to create content on the docs site.